covid_xprize/standard_predictor/tests/test_xprize_predictor.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 (c) Cognizant Digital Business, Evolutionary AI. All rights reserved. Issued under the Apache 2.0 License.
+
+import os
+import unittest
+import urllib.request
+
+import pandas as pd
+
+from covid_xprize.standard_predictor.xprize_predictor import XPrizePredictor
+
+ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+EXAMPLE_INPUT_FILE = os.path.join(ROOT_DIR, "../../validation/data/2020-09-30_historical_ip.csv")
+DATA_FILE = os.path.join(ROOT_DIR, "../data/OxCGRT_latest.csv")
+PREDICTOR_WEIGHTS = os.path.join(ROOT_DIR, "../models/trained_model_weights.h5")
+
+START_DATE = "2020-08-01"
+END_DATE = "2020-08-04"
+
+
+class TestXPrizePredictor(unittest.TestCase):
+
```

Comment:
Keeping the setUpClass method for explicitness, though it does nothing.

Reply:
I would just remove it. It serves no purpose

Reply:
removed 👍